bioimageio_collection_backoffice/remote_resource.py

```diff
@@ -539,6 +539,18 @@ def _set_status(self, value: StagedVersionStatus):
         versions_update = VersionsWithDefaults(staged={self.number: updated_info})
         self.concept.extend_versions(versions_update)
 
+    def lock_publish(self):
+        """Creates publish lock in DB"""
+        self.client.put_and_cache(self.publish_lockfile_path, b" ")
+
+    def unlock_publish(self):
+        """Releases publish lock in DB"""
+        self.client.rm(self.publish_lockfile_path)
+
+    @property
+    def publish_lockfile_path(self):
+        return f"{self.folder}lock-publish"
```

Comment:
this should now lock for the resource "concept" (all versions)
```suggestion
        return f"{self.concept.folder}lock-publish"
```